ingenialink/servo.py

```diff
@@ -41,6 +47,87 @@
 OPERATION_TIME_OUT = -3
 
 
+class DictionaryFactory:
+    """Dictionary factory, creates a dictionary instance depends on the file version
+    and connection interface"""
+
+    VERSION_ABSOLUTE_PATH = "Header/Version"
```

Comment:
I personally ready this as a variable that the user of DictioanryFactory should use.
If it private I would put an underscore, or leave it as a constant of the module (if scoping is correct)